periodictable/nsf.py

```diff
@@ -518,6 +549,7 @@ def init(table, reload=False):
         nsf.is_energy_dependent = (columns[6] == 'E')
         nsf.coherent, nsf.incoherent, nsf.total, nsf.absorption \
             = [fix_number(a) for a in columns[7:]]
+        nsf.b_pp = nsf.absorption/(-2000*ABSORPTION_WAVELENGTH)
```

Comment:
Another hardcoded constant == 2000... would it be worth collecting these as module constants, with comments?

Reply:
Probably not worth collecting. If so, it would be something like:
```
BARN_OVER_ANGSTROM_TO_FM = 1e-3
nsf.b_pp = nsf.absorption/(2*ABSORPTION_WAVELENGTH)*BARN_OVER_ANGSTROM_TO_FM
```